roles/openshift_logging/tasks/delete_logging.yaml

```diff
@@ -95,3 +95,36 @@
     - logging-elasticsearch
     - logging-fluentd
     - logging-mux
+
+# delete eventrouter
```

Comment:
We should move these to the role and figure out how to include them from the parent logging role as a task

Reply:
Here I would also feel more comfortable with keeping the changes related to the introduction of eventrouter only and do refactoring in a separate PR. Currently, no role has its own uninstall.

Reply:
But we have already created a new role.  It seems its simple enough to add a file to it for deleting the event router and include or call the task here.